matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/mapper/EventMapper.kt

```diff
@@ -38,6 +39,7 @@ internal object EventMapper {
         eventEntity.content = ContentMapper.map(event.content)
         val resolvedPrevContent = event.prevContent ?: event.unsignedData?.prevContent
         eventEntity.prevContent = ContentMapper.map(resolvedPrevContent)
+        eventEntity.isUseless = event.type == EventType.STATE_ROOM_MEMBER && eventEntity.content == eventEntity.prevContent
```

Comment:
I don't think it should be saved in db?

Reply:
It's saved in DB to be able to do query on it. Maybe you known a better way to do query which check if two value in the DB are identical?

Reply:
Oh, right, thanks to realm for not being flexible enough...
Could you create a class like IsUselessResolver to handle this?

Reply:
I thought about it, but, it will call again what is called in the three line above, so for the perf, as we are inserting event in DB here, also from initial sync, I do not know if it's a good idea... What do you think?

Reply:
Ok done, I let you check